libs/net/entities/ReportInstanceContent.cs

```diff
@@ -37,6 +37,7 @@ public class ReportInstanceContent : AuditColumns
     /// <summary>
     /// get/set - The name of the section this content belongs.
     /// </summary>
+    [Column("section_name")]
```

Comment:
Regrettably forgot this in the last PR and DB migration...  Would love it if anyone did code review.